scripts/install

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+set -e
+
+TAG_PROTOTYPE_CJDNS_PI2=autorun
+TAG_CJDNS=cjdns-v17.3
```

Comment:
@JoshuaOpolko Though the issue we encountered is fixed on crashey, let's stay on a tag so we never compile against changing code again. **v17.3** is the latest stable.